sections/semantics-scriptings.include

```diff
@@ -2094,71 +2124,165 @@
-  </dd>
-
-  <dt>
-    <a>Allowed ARIA state and property attributes</a>:
-  </dt>
-  <dd><a>Global aria-* attributes</a></dd>
-  <dd>
-    Any <code>aria-*</code> attributes
-    <a href="#allowed-aria-roles-states-and-properties">applicable to the allowed roles</a>.
-  </dd>
-
-  <dt>
-    <a>DOM interface</a>:
-  </dt>
-
-  <dd>Supplied by the element's author (inherits from {{HTMLElement}})</dd>
-</dl>
+  <dl class="element">
+    <dt>
+      <a>Categories</a>:
+    </dt>
+    <dd><a>Flow content</a>.</dd>
+    <dd><a>Phrasing content</a>.</dd>
+    <dd><a>Palpable content</a>.</dd>
+
+    <dt>
+      <a>Contexts in which this element can be used</a>:
+    </dt>
+    <dd>
+      Where <a>phrasing content</a> is expected.
+    </dd>
+
+    <dt>
+      <a>Content model</a>:
+    </dt>
+    <dd>
+      <a>Transparent</a>.
+    </dd>
+
+    <dt>
+      <a>Content attributes</a>:
+    </dt>
+    <dd>
+      <a>Global attributes</a>, except the <{global/is}> attribute
+    </dd>
+
+    <dt>
+      <a>Allowed ARIA role attribute values</a>:
+    </dt>
+    <dd>
+      <a href="#allowed-aria-roles-states-and-properties">Any role value</a>
+    </dd>
+
+    <dt>
+      <a>Allowed ARIA state and property attributes</a>:
+    </dt>
+    <dd><a>Global aria-* attributes</a></dd>
+    <dd>
+      Any <code>aria-*</code> attributes
+      <a href="#allowed-aria-roles-states-and-properties">applicable to the allowed roles</a>.
+    </dd>
+
+    <dt>
+      <a>DOM interface</a>:
+    </dt>
+    <dd>
+      Supplied by the element's author (inherits from {{HTMLElement}})
+    </dd>
+  </dl>
 
   An <a>autonomous custom element</a>'s meaning is defined by its author.
   It <a>represents</a> its children.
 
+  <div class="example">
+    To illustrate how to create an <a>autonomous custom element</a>, the following example
+    will create the foundation for a custom heading element. The element will have two required
+    attributes (<code>level</code> and <code>text</code>) to pass in the heading level and text
+    for the heading. The permalink will be created based on the text of the heading.
+
+
+    Before an author can use a custom element in their markup, a class for the new element must be
+    declared by extending {{HTMLElement}} and defining the {{constructor}}. The class always
+    starts by calling <code>super()</code> so that the correct prototype chain will be
+    established.
+
+    <xmp highlight="javascript">
+      class permaHeading extends HTMLElement {
+        constructor() {
+          super();
+          // Element functionality will go here
+        }
+      }
+    </xmp>
+
+    All of the custom element's functionality will be written within the {{constructor}}. It's here
+    where the custom element's shadow DOM structure and styling may be defined and attached to
+    the custom element's <a>shadow root</a>.
+
+    The basic functionality for the custom element is defined in the following JavaScript. The
```

Comment:
maybe emphasize more that this is base-level functionality, as there's definitely a bit more that could be done here to make it into a fully-realized reusable component.